src/server/saved_objects/routes/find.js

```diff
@@ -31,6 +31,7 @@ export const createFindRoute = (prereqs) => ({
         page: Joi.number().min(0).default(1),
         type: Joi.array().items(Joi.string()).single().required(),
         search: Joi.string().allow('').optional(),
+        search_operator: Joi.string().allow('OR', 'AND').optional(),
```

Comment:
You probably want `Joi.string().valid('OR', 'AND')`, which will limit the options, while `.allow()` adds _additional_ values to the values supported by `Joi.string()`. 

I also think this should use `.default('OR')` instead of using `.optional()`.